runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/layouts/metrics/HistogramsLayout.java

```diff
@@ -166,11 +169,17 @@ public Builder mode(
         public HistogramsLayout build()
         {
             final File layoutFile = path.toFile();
+            MappedByteBuffer mappedBuffer;
             if (mode == Mode.CREATE_READ_WRITE)
             {
                 CloseHelper.close(createEmptyFile(layoutFile, capacity));
+                mappedBuffer = mapExistingFile(layoutFile, HISTOGRAMS_LABEL);
+            }
+            else
+            {
+                assert mode == Mode.READ_ONLY;
+                mappedBuffer = mapExistingFile(layoutFile, READ_ONLY, HISTOGRAMS_LABEL);
```

Comment:
The `mappedBuffer` assignment can happen outside the `if-else` passing `mode.mapMode()` and avoiding the repeated code, agree?

Btw, do we actually need `Mode` instead of `boolean readonly` like the other layouts?

Reply:
done